.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a bug report
+title: ''
+labels: bug, needs triage
+assignees: ''
+
+---
+
+**Description:**
+A clear and concise description of what the bug is.
+
+**Task version:**
+Specify the task version
```

Comment:
I would stick with `action` version. Task is the term used in ADO that we don't carry over

Reply:
Done. Thank you.